README.md

```diff
@@ -61,6 +61,20 @@ To activate Poetry's virtual environment, run:
 
     poetry shell
 
+### Setting up Wagtail guide to develop on Docker
+
+1. Create a `.env` file in the project root containing these variables, you can adjust the values to your preferences:
+
+    ALLOWED_HOSTS=localhost
+    PORT=8000
+    SECRET_KEY=some-random-secret
+    DJANGO_SETTINGS_MODULE=apps.guide.settings.dev
+
+2. Build the image by running the `make docker-build` command.
+3. Run the container with `make docker-run`.
+4. Run the init script in the container: `make docker-init`
+5. You should now have access to the project in your browser at `http://localhost:8000`
+
```

Comment:
This will need to have prettier run on it. `make format`

Reply:
I'm having some trouble with my yarn installation, but I ran it through preview mode on Github and made some changes

Reply:
The lint failure could be other code sorry. 

https://github.com/wagtail/guide/pull/173